selfswab/models.py

```diff
@@ -0,0 +1,98 @@
+import uuid
+
+from django.db import models
+from django.utils import timezone
+
+from userprofile.validators import za_phone_number
+
+
+class SelfSwabScreen(models.Model):
+    LOW_RISK = "Low"
+    HIGH_RISK = "High"
+    RISK_TYPES = ((LOW_RISK, "Low"), (HIGH_RISK, "High"))
+
+    AGE_U18 = "<18"
+    AGE_18T40 = "18-40"
+    AGE_40T65 = "40-65"
+    AGE_O65 = ">65"
+    AGE_CHOICES = (
+        (AGE_U18, AGE_U18),
+        (AGE_18T40, AGE_18T40),
+        (AGE_40T65, AGE_40T65),
+        (AGE_O65, AGE_O65),
+    )
+
+    GENDER_MALE = "male"
+    GENDER_FEMALE = "female"
+    GENDER_OTHER = "other"
+    GENDER_NOT_SAY = "not_say"
+    GENDER_CHOICES = (
+        (GENDER_MALE, "Male"),
+        (GENDER_FEMALE, "Female"),
+        (GENDER_OTHER, "Other"),
+        (GENDER_NOT_SAY, "Rather not say"),
+    )
+
+    REGISTRATION_INCOMPLETE = "incomplete"
+    REGISTRATION_COMPLETE = "complete"
+    REGISTRATION_TYPES = (
+        (REGISTRATION_INCOMPLETE, "incomplete"),
+        (REGISTRATION_COMPLETE, "complete"),
+    )
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    created_by = models.CharField(max_length=255, blank=True, default="")
+    contact_id = models.CharField(max_length=255, blank=False)
+    msisdn = models.CharField(
+        max_length=255, validators=[za_phone_number], db_index=True
+    )
+    registration = models.CharField(
```

Comment:
I think we should remove this for now, we won't be sending incomplete registrations to this api. Mostly because I'm not sure when we will do the post.

Reply:
will do